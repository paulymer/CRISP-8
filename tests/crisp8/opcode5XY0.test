`¯a¯b®Pÿÿ’0ÿÿ