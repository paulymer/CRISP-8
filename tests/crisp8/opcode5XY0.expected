PC: 0x0111  I: 0x0000
V0: 175  V1: 175  V2: 174  V3:   0  V4:   0  V5:   0  V6:   0  V7:   0
V8:   0  V9:   0  Va:   0  Vb:   0  Vc:   0  Vd:   0  Ve:   0  Vf:   0
0200: 60af 61af 62ae 5010 ffff 9230 ffff 1111